packages/catalog-search/src/SearchSuggestions.jsx

```diff
@@ -0,0 +1,62 @@
+import { Link } from 'react-router-dom';
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const SearchSuggestions = ({ autoCompleteHits, enterpriseSlug, handleViewAllClick }) => (
+  <div className="suggestions" data-testid="suggestions">
+    <div>
+      <div className="mb-2 ml-2 mt-1 font-weight-bold suggestions-section">
+        Courses
+      </div>
+      {
+                autoCompleteHits.filter(hit => hit.content_type === 'course')
+                  .slice(0, 3)
+                  .map((hit) => (
+                    <Link to={`/${enterpriseSlug}/course/${hit.key}`} key={hit.title} className="suggestion-item" style={{ whiteSpace: 'pre-wrap' }}>
```

Comment:
I think the code will a lot more elegant if we pass a reference to a function here that will return us the path:

```javascript
<Link
   key={hit.title}
   className="suggestion-item"
   style={{ whiteSpace: 'pre-wrap' }}
   to={linkToCourse}
 >

```

Reply:
done